data/json/recipes/recipe_food.json

```
@@ -1907,7 +1907,7 @@
     "time": "1 h",
     "autolearn": true,
     "batch_time_factors": [ 50, 5 ],
-    "book_learn": [ [ "survival_book", 3 ], [ "textbook_survival", 3 ], [ "manual_survival", 3 ] ],
+    "book_learn": [ [ "survival_book", 3 ], [ "textbook_survival", 3 ], [ "manual_survival", 3 ], [ "dairy_book", 3 ] ],
```

```suggestion
    "book_learn": [ [ "survival_book", 7 ], [ "dairy_book", 3 ] ],
```
I'm not buying there being a hard cheese recipe in a book called "pitching a tent", or a book for experienced hikers. Diary of a mountain man, sure, but it's going to be hard to intepret from reading an autobiography.